makefile

```diff
@@ -12,15 +12,12 @@ help:
 # ----------------------------------------------------------------------------------------------------------------
 
 install: ## Installs all dependencies
-	curl -O https://orca-build.io/downloads/orca.zip
-	unzip -o orca.zip
-	rm -f orca.zip
 	curl -L https://www.svrunit.com/downloads/svrunit.zip --output svrunit.zip
 	unzip -o svrunit.zip
 	rm -f svrunit.zip
 
 generate: ## Generates all artifacts for this image
-	@php orca.phar --directory=.
+	docker run -v ${PWD}:/opt/project orcabuilder/orca:latest
```

Comment:
hi
thanks for this
the problem with this is, i found out that it takes way longer than the local version.
so i did not yet push it.
though it might be the only way if you dont have a local php right? ;)

Reply:
exactly!

Reply:
After testing in different environments, I'm guessing you're on a Mac?
Mac is known to be slow with Docker volumes.
Any other environment should be fine.
Generation[ works perfectly fine in a Linux environment](https://streamable.com/h2qy55).

Reply:
haha yes, i know,
but at the moment we where the only one generating it, so i liked that comfort haha
thanks for the PR :)

Reply:
codespaces is free for organisations at the moment 😉  might be worth a try